routes/conversation.js

```diff
@@ -32,6 +32,11 @@ exports.create = function (req, res){
 		var people = idOrEmails.filter(function(idOrEmail){return idOrEmail.indexOf('@') === -1});
 		var emails = idOrEmails.filter(function(idOrEmail){return idOrEmail.indexOf('@') !== -1});
 
+		// Limit from 2 - 6 people, including oneself.
+		if (idOrEmails.length < 1 || idOrEmails.length > 5){
+			resError(res, "Keep it intimate!  Please have 2 - 6 people in a conversation.");
```

Comment:
I'm nitpicking, but I'd swap out the "-" in "2 - 6 people" with a to:
"2 to 6"
So that it's more human readable.